command/plan.go

```diff
@@ -129,12 +112,18 @@ func (c *PlanCommand) Run(args []string) int {
 	}
 
 	// Force the region to be that of the job.
-	if r := job.Region; r != "" {
+	if r := *job.Region; r != "" {
 		client.SetRegion(r)
 	}
 
+	// Check that the job is valid
+	if _, err := client.Jobs().Validate(job, nil); err != nil {
+		c.Ui.Error(fmt.Sprintf("Error validating job: %s", err))
+		return 255
+	}
+
 	// Submit the job
-	resp, _, err := client.Jobs().Plan(apiJob, diff, nil)
+	resp, _, err := client.Jobs().Plan(job, diff, nil)
```

Comment:
Can Plan just validate the job and then you don't need two RPCs. I think it already does